src/components/CardModal/CardModal.js

```diff
@@ -13,7 +13,7 @@ const CardModal = ({ card }) => {
   const [moveOpen, setMoveOpen] = useState(false);
   const [selectedListId, setSelectedListId] = useState("");
   const [editableContent, setEditableContent] = useState(card.content);
-  const [width, setWidth] = useState(20);
+  const [rows, setRows] = useState(1);
```

Comment:
The input element makes use of width, but for the text area, I've just set this to w-full below.
This state has been changed to control the amount of rows a text area has instead.
Rows correspond (roughly) to the height of a textarea based on sentences i.e.
If rows is 1, the height is set such that only one sentence is shown and you have to scroll to see the rest